python/paddle/v2/data_feeder.py

```diff
@@ -75,6 +75,13 @@ def __init__(self, data_types, feeding=None):
         input_types = []
         if feeding is None:
             feeding = default_feeding_map(data_types)
+        elif isinstance(feeding, collections.Sequence):
```

Comment:
As here we allow `feeding` to be a new type -- Sequence, we should update/add the example in above comment?